src/internal/EnhancedButton.js

```diff
@@ -296,7 +296,7 @@ class EnhancedButton extends Component {
       display: 'inline-block',
       fontFamily: this.context.muiTheme.baseTheme.fontFamily,
       WebkitTapHighlightColor: enhancedButton.tapHighlightColor, // Remove mobile color flashing (deprecated)
-      cursor: disabled ? 'default' : 'pointer',
+      cursor: disabled ? 'not-allowed' : 'pointer',
```

Comment:
But `< EnhancedButton />` component did manage some basic own styles.
Why do we need this component at all? My thoughts was about common abstraction for all elements with similar behavior.
Moreover, I think that `cursor: not-allowed` is bigger than just visual helper. It is like 'metadata' for user that there is no interaction with this element.